maintenance/image-options/alternate-registry.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Configure use of your image registry 
+description: Configure Calico to pull images from a public or private registry. 
+canonical_url: '/maintenance/image-options/alternate-registry'
+---
+
+### Big picture
+
+Configure {{site.prodname}} to pull images from a registry (public or private).
+
+### Value
+
+In many deployments, installing {{site.prodname}} in clusters from third-party private repos is not an option. {{site.prodname}} offers these public and private registery options, which can be used in any combination: 
+
+- **Install from a registry** for use cases like airgapped clusters, or clusters with bandwidth or security constraints
+- **Install from an image path in a registry** if you have pulled {{site.prodname}} images to a sub path in your registry
+- [Install images by registry digest]({site.baseurl}}/maintenance/image-options/imageset)
```

Comment:
typo: `{site.baseurl}}` should be `{{site.baseurl}}`

Reply:
fixed.